### PR TITLE
In 151735125 get race ethnicity with person

### PR DESCRIPTION
**Pull request by ThomasOSI**
### Pivotal Story

- [**HOTLINE: E2E ** Display Race for existing Person added to a Screening](https://www.pivotaltracker.com/story/show/151735125)

### Purpose
When adding a person to a screening from the search, bring any race and ethnicity information that's available from legacy

### Background
It was a dark and windy night...

### Notes for Reviewer
This story does not include showing the info in the search results, just the resulting PoS card

### CODE REVIEW

File: app/models/ethnicity.rb [hunk 1]
```diff
@@ -5,5 +5,5 @@ class Ethnicity
   include Virtus.model
 
   attribute :hispanic_latino_origin, String
-  attribute :ethnicity_detail, String
+  attribute :ethnicity_detail, Array
```
Comment (AmanShurafa) [inline app/models/ethnicity.rb]: ethnicity_detail should be plural since it's more than 1. But I leave it up to you to decide since it's named this way at the front end, api, and ferb api.

File: app/helpers/race_helper.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module RaceHelper # :nodoc:
+  def self.best_matches_by_race(races)
+    keys = races.uniq { |race| race['race'] }
```
Comment (jgedeon-osi) [inline app/helpers/race_helper.rb]: Can we call race item instead like you do below.

File: spec/controllers/api/v1/participants_controller_spec.rb [hunk 3]
```diff
@@ -83,8 +84,9 @@
     let(:participant_params) do
       {
         id: '1',
-        first_name: 'Marge',
+        first_name: 'Margie',
```
Comment (AmanShurafa) [inline spec/controllers/api/v1/participants_controller_spec.rb]: Why was this changed?
